Parking/ParkingGroup/doc/spec.md

```diff
@@ -59,6 +59,24 @@ per group type.
     + Normative References: [https://schema.org/description](https://schema.org/description)
     + Optional
 
++ `status` : Status of the parking group.
+    + Attribute type: List of [Text](http://schema.org/Text)
+    + Metadata:
+        + `timestamp` : Timestamp of the last attribute update
+        + Type: [DateTime](https://schema.org/DateTime)
+    + Allowed values: The following defined by the following enumerations defined by DATEX II version 2.3 :
+        + *ParkingSiteStatusEnum*
+        + *OpeningStatusEnum*
+        + (`open`, `closed`, `closedAbnormal`,`openingTimesInForce`, `full`,
+           `fullAtEntrance`, `spacesAvailable`, `almostFull`)
+        + Or any other application-specific
+    + Optional
+
++ `floorNumber` : For parking sites with multiple floor levels, floor of this group.
```

Comment:
be careful with the naming. and the specification. Are you referring to the number of floors or the to the floor level? 

I would suggest to name this attribute just `floor` . But not only that, you need to specify clearly that ground level is represented by `0`, underground levels are negative integer numbers starting with `-1` and aboveground levels are positive integer numbers starting with 1.